### PR TITLE
Change the URL for the Issue

### PR DESCRIPTION
**Pull request by mikevhe18**

### CODE REVIEW

**Comment by mikevhe18** [comment]
@pedrobaeza As this module is moved from the OCA / website, I change the URL for Issue in Bug Tracker

File: website_membership_contact_visibility/__openerp__.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
 # © 2016 Michael Viriyananda
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Website Membership Contact Visibility',
     'version': '8.0.1.0.0',
+    'summary': 'Adding functionality to set the visibility of contact information of each member',
```
Comment (pedrobaeza) [inline website_membership_contact_visibility/__openerp__.py]: This line is longer than 80 cols. Split it across several lines.

**Comment by mikevhe18** [comment]
@pedrobaeza I've made the changes.

**Comment by pedrobaeza** [comment]
Can you please squash all the commits, because they are very little changes?

**Comment by mikevhe18** [comment]
@pedrobaeza rebased and squash

**Comment by pedrobaeza** [comment]
Thanks,

Fast-tracking.

**Pull request merged**